.github/workflows/nextflow-tests.yml

```diff
@@ -4,12 +4,100 @@ name: 'Nextflow tests'
 on:
   workflow_call:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.sha }}
+  cancel-in-progress: true
+
 # Inspired by https://blog.aspect.dev/github-actions-dynamic-matrix
 jobs:
   discover:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/github-script@v7
+        name: Parse event
+        id: parse-event
+        with:
+          script: |
+            /*
+            We need to determine if the incoming event was from a pull_request
+            or an issue_comment
+            */
+            let apply_fixes = context.eventName == 'issue_comment'
+
+            let pr = {}
+
+            let abort_early = false
+
+            if (apply_fixes) {
+              // Actions bot ID - see https://api.github.com/users/github-actions%5Bbot%5D
+              let bot_id = 41898282
+              let found_review = false
+
+              // Make sure that changes are actually requested for this PR
+              for await (const response of github.paginate.iterator(
+                github.rest.pulls.listReviews,
+                {
+                  owner: context.repo.owner,
+                  repo: context.repo.repo,
+                  pull_number: context.payload.issue.number
+                }
+              )) {
+                if (found_review) break
+
+                for (const review of response.data) {
+                  if (review.user.id == bot_id && review.state == "CHANGES_REQUESTED") {
+                    found_review = true
+                    break
+                  }
+                }
+              }
+
+              if (!found_review) {
```

Comment:
I suggest inverting if/else -
```javascript
if (found_review) {}
else {}
```
vs
```javascript
if (!found_review) {}
else {}
```

In my experience, reading the code tends to flow more naturally, requiring less cognitive load to parse the initial meaning. However, I know this can be subjective.

Reply:
Done with 64f874d.